osu.Game/Utils/SentryLogger.cs

```diff
@@ -7,26 +7,26 @@
 using System.Net;
 using System.Threading.Tasks;
 using osu.Framework.Logging;
-using SharpRaven;
-using SharpRaven.Data;
+using Sentry;
 
 namespace osu.Game.Utils
 {
     /// <summary>
     /// Report errors to sentry.
     /// </summary>
-    public class RavenLogger : IDisposable
+    public class SentryLogger : IDisposable
     {
-        private readonly RavenClient raven = new RavenClient("https://5e342cd55f294edebdc9ad604d28bbd3@sentry.io/1255255");
+        private IDisposable sentry;
```

Comment:
Weird API choice *to have `IDisposable`* as the returned type, but alas...

Reply:
There is a good sense `SentryClient`, but missing `Breadcrumb` support.
An alternative is use a permanent scope.